ui/src/app/pages/signed-in/inactivity-monitor.tsx

```diff
@@ -154,7 +161,7 @@ export const InactivityMonitor = () => {
       });
     };
 
-    signOutIfLocalStorageInactivityElapsed('/sign-in-again');
+    signOutIfLocalStorageInactivityElapsed('/login');
```

Comment:
why change this?

Reply:
There's a bug in the current `signOut` implementation where `continuePath` is only used when the logout request fails. As a result, we were rarely visiting `/sign-in-again`. I updated the `signOut` function to use `continuePath` in all cases.

However, the result of this is that were now being redirected to `/sign-in-again` even when the user revisits the site hours after ending their session.

This particular line is called when the component first loads. So if the user loads the page and is "inactive," they are redirected to the homepage instead.

If you think it's better, we could instead show them the `/sign-in-again` page here. However, I think going back to the login page is less jarring and typical of other websites with short session expirations (like banks).

I described these differences in the [difference tracking doc](https://docs.google.com/document/d/1A_DN49DY5Qrrq2ZwlK8t9GrjG253GursFfS2trR6t-s/edit#heading=h.jazmc3rqeh26). CTRL-F for "inactivity".

Reply:
I am not clear on the intentions of the "sign-in again" page, but this explanation looks fine to me